README.md

```diff
@@ -8,7 +8,7 @@ Backend for the [Emote Widget](https://github.com/fostive/emote-widget)
 
 ## Installation & Deployment
 
-[![Deploy](https://www.herokucdn.com/deploy/button.svg)](https://heroku.com/deploy)
+[![Deploy](https://www.herokucdn.com/deploy/button.svg)](https://heroku.com/deploy?template=https://github.com/fostive/emote-widget/tree/main)
```

Comment:
I think this should be `emote-server` instead of `emote-widget`?

Reply:
Oh you are right!!

Reply:
@crcastle done!